src/streamlink_cli/main.py

```diff
@@ -41,12 +43,14 @@
 
 args = console = streamlink = plugin = stream_fd = output = None
 
+log = logging.getLogger("streamlink.console")
```

Comment:
This changes the main log output from `[cli][foo]bar` to `[console][foo]bar`, which breaks the Twitch GUI's error handling.
I had to fix this in the Twitch GUI, but since there's no new release yet, everyone who's updated Streamlink will run into potential issues with that. Same for other applications relying on the log output, ofc. Unfortunately, I didn't notice this earlier and I should have commented on this sooner.

Reply:
@beardypig Is there are reason this was changed from "cli" to "console"?

Reply:
Nah, was a mistake

Reply:
Can we revert this and publish a bugfix release, please? :confused:

Reply:
If you’re able to do it now you can, I won’t be able to do it until tomorrow - don’t have access to a PC right now...